README.md

```diff
@@ -502,6 +503,37 @@ Supported error codes:
 * `pb_utils.TritonError.UNAVAILABLE`
 * `pb_utils.TritonError.UNSUPPORTED`
 * `pb_utils.TritonError.ALREADY_EXISTS`
+* `pb_utils.TritonError.CANCELLED` (since 23.10)
+
+#### Request Cancellation Handling
+
+One or more requests may be cancelled by the client during execution. Starting
+from 23.10, `request.is_cancelled()` returns whether the request is cancelled.
+
+If a request is cancelled, the model may respond with any dummy object in place
+of the normal output tensors on the request. For example:
+
+```python
+import triton_python_backend_utils as pb_utils
+
+class TritonPythonModel:
+    ...
+
+    def execute(self, requests):
+        responses = []
+
+        for request in requests:
+            if request.is_cancelled():
+                responses.append(None)
```

Comment:
I believe `append(None)` would not work. You would need to create an explicit response object (e.g., `InferenceResponse(error=TritonError(CANCELLED))`)

Reply:
Updated, as the logic is reverted: [Rollback non-decoupled any response on cancel](https://github.com/triton-inference-server/python_backend/pull/304/commits/3de5922c48ece05b5d55862063e67ab3a22c97f5)